Source/Diagnostics/FlushFormats/FlushFormatOpenPMD.H

```diff
@@ -53,6 +53,7 @@ public:
 private:
     /** This is responsible for dumping to file */
     std::unique_ptr< WarpXOpenPMDPlot > m_OpenPMDPlotWriter;
+    int m_NumAggBTDBufferToFlush=5;
```

Comment:
- Docstring needed, please.
- Formatting (pls see code around it).
```suggestion
    /** Please add a descriptive doc string here, Junmin. */
    int m_NumAggBTDBufferToFlush = 5;
```

Reply:
added in the next commit